code/modules/mob/living/simple_animal/abnormality/aleph/mountain.dm

```diff
@@ -14,9 +14,11 @@
 	melee_damage_upper = 30
 	melee_damage_type = RED_DAMAGE
 	armortype = RED_DAMAGE
+	blood_volume = 0
```

Comment:
This will cause bullets and weapons to create no blood effect, which just looks bad.

Reply:
He makes his own blood on hit now

Reply:
Unless you mean the "splatter" effect?